contrib/ubase/patches/watch.patch

```diff
@@ -0,0 +1,31 @@
+Index: ubase/ubase/watch.c
+===================================================================
+--- ubase.orig/ubase/watch.c	2023-01-01 22:32:22.325582802 +0100
++++ ubase/ubase/watch.c	2023-01-01 22:33:42.278913458 +0100
+@@ -17,7 +17,7 @@
+ {
+ 	char cmd[BUFSIZ];
+ 	char *end;
+-	useconds_t interval = 2 * 1E6;
```

Comment:
[useconds_t](http://bxr.su/xref/NetBSD/sys/sys/types.h#324) should be defined in `sys/types.h` as `unsigned int`.

Reply:
It seems to me, that it's better to have `interval` here as `unsigned int`, since we've decided to use `sleep()` (which takes `unsigned int` as an argument) instead of `usleep()` (which takes `useconds_t`). If we left `useconds_t` here, it would indicate that `interval` is a value in microseconds, which could be misleading.

Reply:
I agree! This makes sense.